app/views/moderations/modals/_unpublish_all_posts.html.erb

```diff
@@ -5,6 +5,7 @@
   <p class="mt-2">
```

Comment:
These 2 modals are very similar, but are "submitted" differently. I haven't done any refactoring at this point.
Overall, I would at least use the same locales :thinking: